content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+### 4. Candy Machine Configuration
+
+Now that we have installed Metaplex, it's time to configure our candy machine.
+
+Open the `config.json` located in our metaplex-starter folder. Copy and paste the configurations below into `config.json`:
+
+```json
+{
+    "price": 1,
+    "number": 5,
+    "gatekeeper": null,
+    "solTreasuryAccount": "<YOUR_WALLET_ADDRESS>",
+    "splTokenAccount": null,
+    "splToken": null,
+    "goLiveDate": "25 Dec 2020 00:00:00 GMT",
+    "endSettings": null,
+    "whitelistMintSettings": null,
+    "hiddenSettings": null,
+    "storage": "arweave",
+    "ipfsInfuraProjectId": null,
+    "ipfsInfuraSecret": null,
+    "awsS3Bucket": null,
+    "nftStorageKey": null,
+    "noRetainAuthority": false,
+    "noMutable": false
+}
+```
+
+Update the `solTreasuryAccount` field with the wallet address we imported to Phantom. This will be the only setting we change for now.
+
+Let's walk through the values we have real quick:
+- `price` is set to `1`, meaning each NFT will cost 1 SOL.
+- `number` is set to `5`, meaning the collection has 5 NFTs total.
+- `goLiveDate` is set to a date in the past so that the collection will be live as soon as we upload it.
+- `storage` is set to `arweave` which means the metadata will be stored for 7 days on arweave. Rembember, if we were doing this for Mainnet we would use `arweave-sol` to make the storage permanent.
+- `noRetainAuthority` is set to `false` so that minter of the NFT cannot change the metadata.
+- `noMutable` is set to `false` so that we can update the metadata after the fact if we need to.
+
+### 5. Prepare Assets
+
+Now let's prepare the assets for our candy machine. The starter code includes an `assets` folder that holds the images and the metadata files we'll use for our collection. 
+
+Open the `0.json` file. Notice that all of the fields are already filled out except for the `address` field under `creators`. You'll need to set this to the address we imported to Phantom. Do this for the remaining `json` files in the `assets` folder.
+
+```json
+{
+    "name": "1",
+    "symbol": "TEST",
+    "image": "0.png",
+    "properties": {
+        "files": [{ "uri": "0.png", "type": "image/png" }],
+        "category": "image",
+        "creators": [
+            {
+                "address": "<YOUR_WALLET_ADDRESS>",
+                "share": 100
+            }
+        ]
+    },
+    "description": "",
+    "seller_fee_basis_points": 500,
+    "attributes": [
+        { "trait_type": "background", "value": "blue" },
+        { "trait_type": "eyes", "value": "star-eyes" },
+        { "trait_type": "mouth", "value": "triangle-mouth" },
+        { "trait_type": "face", "value": "teal-face" }
+    ],
+    "collection": { "name": "NAME", "family": "FAMILY" }
+}
+```
+
+Once you’ve updated the `address` field for each `.json` file, we can verify that the assets are ready for upload by running Candy Machine's `verify_assets` command.
+
+This will check that:
```

Comment:
@jamesrp13 worth reiterating the command name? i.e.

The `verify_assets` command will check that: